### ISSUE TITLE
Midterm - Water

### ISSUE THREAD

**Issue by ProfPollati** [Bloomfield-FA15-CAT238/Midterm-KeaunRobinson#4]
Water scaled to 0.00314,0,0.00351 is essentially not having it in the game. With a scale of zero, it is not going to show up at all!

**Comment by Keaun** [comment]
Umm did the water have to be moving and animated? I imported the water and
added the texture but it wasn't moving for me and I couldn't see any
difference in scaling it down. Can you scale a terrain down or no?

**Comment by ProfPollati** [comment]
No, the water had to be visible. You should have added the Prefab in **Standard Assets -> Environment -> Water -> Water -> Prefabs** or  **Standard Assets -> Environment -> Water -> Water4 -> Prefabs**
